src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue

```diff
@@ -51,7 +52,7 @@
 			</slot>
 		</span>
 
-		<span :class="['checkbox-content__text', textClass]">
+		<span v-if="$slots.default" :class="['checkbox-content__text', textClass]">
```

Comment:
I think this is not necessary (although it probably doesn't hurt either), since we have `display: none;` already if the span is empty:
https://github.com/nextcloud-libraries/nextcloud-vue/blob/8aec0a1f1848efb229401e2796537ba9fad8d449/src/components/NcCheckboxRadioSwitch/NcCheckboxContent.vue#L225-L228

Or do I miss something here?

Reply:
Just cleaner approach in general to really not have an extra unnecessary element.
It can quickly adds up in lists (like Files) :)

Reply:
I'd call it a good practice to always check for slots on slot wrapper elements.